styles/main.css

```diff
@@ -0,0 +1,243 @@
+/**** External Imports *****/
+
+@import "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.7.2/font/bootstrap-icons.css";
+
+/***** Defining Color Varibles *****/
+:root {
+  --primary-color: #ff3b30;
+  --secondary-color: #fff6ea;
+  --text-color-black: #2b2b2b;
+  --body-background-color: #ffffff;
+  --dark-white: #f7f7f7;
+  --item-background-color: #c5c5c5;
+  --hover-color: #ef4444;
+  --box-shadow-down: 0px 0px 10px 4px rgb(0 0 0 / 5%);
+  --box-shadow-up: 0px -4px 10px rgb(0 0 0 / 5%);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 1rem;
+  background-color: var(--body-background-color);
+  color: var(--text-color-black);
+  box-shadow: var(--box-shadow-down);
+}
+
+.small-text {
+  font-size: 0.6rem;
```

Comment:
use px unit when value is less than 1rem

Reply:
Since px was not throwing a satisfying result so I used this. I have also taken the advice from fellow members.

Reply:
Rem is good choice